src/rm.js

```diff
@@ -43,6 +44,10 @@ function rmdirSyncRecursive(dir, force) {
     }
   }
 
+  // if was directory was referenced through a symbolic link,
+  // do not remove the directory itself
+  if (fromSymlink) return;
```

Comment:
Can you specify that while we do not remove the directory itself, we do in fact remove its contents?